packages/nimble-components/src/table-column/date-text/tests/table-column-date-text.spec.ts

```diff
@@ -78,32 +78,41 @@ describe('TableColumnDateText', () => {
             expect(column.checkValidity()).toBeTrue();
```

Comment:
I pulled your branch and verified that the tests in this file pass on macOS 14.3 in Safari 17.3 and Playwright's Webkit browser.